src/Symfony/Component/ObjectMapper/Attributes/Map.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper\Attributes;
```

Comment:
Minor comment about `Attributes` namespace, lets change to `Attribute` for similarity with other components.